README.md

```diff
@@ -1 +1,83 @@
-# data-structures
\ No newline at end of file
+# data-structures
```

Comment:
It's great that you're showing your code coverage in your README. Truly, it is. But your README is otherwise useless for someone who wants to actually *use* your code moving forward. Describe the modules encapsulated by your package